app/upgrades/v7/upgrades.go

```diff
@@ -49,38 +49,73 @@ func auctionInitGenesis(ctx sdk.Context, auctionKeeper auctionkeeper.Keeper) {
 	genesisState := auctiontypes.DefaultGenesisState()
 
 	usomm52WeekLow := sdk.MustNewDecFromStr("0.079151")
+	eth52WeekHigh := sdk.MustNewDecFromStr("2359.89")
```

Comment:
I am wondering if we about multiply these prices by 1.5 given market conditions

Reply:
We could, 52 week high is an arbitrary choice. At the same time, since prices are denominated in SOMM and we are choosing SOMM's 52 week low, it should have the effect of overpricing in most market conditions. But perhaps adding a flat multiplier on top of it as a param or something would be a good move. We can also control when these prices time out with that auction parameter so they'll have to be updated every N blocks.